.eslintrc.js

```diff
@@ -4,6 +4,7 @@ require('@uniswap/eslint-config/load')
 
 module.exports = {
   extends: '@uniswap/eslint-config/react',
+  ignorePatterns: ['src/utils/__generated__/'],
```

Comment:
nit: big oof that generated files are linted at all. If you're up for it, I think that `@uniswap/eslint-config` should ignore any generated file: `**/__generated__/`. If you're not up for it, could you ping me (in Slack) and I can own or ticket it for the near future?

Reply:
yeah np, I can update eslint-config